manuscript.md

```diff
@@ -78,6 +99,10 @@ Benchmark of PRNG
 
 **Security Analysis**
 
+wyhash is designed for speed and is not to be cryptographically secure. Analysis by @leo-yuriev highlighted that wyhash uses a so-called "narrow-pipe" [Merkle-Dåmgard construction](https://en.wikipedia.org/wiki/Merkle%E2%80%93Damg%C3%A5rd_construction) where manipulation of the input data can lead to entropy loss.
```

Comment:
Suggest:

"wyhash is designed for speed - not to be cryptographically secure."